### ISSUE TITLE
Need to install ember globally for 'ember serve' to work

### ISSUE THREAD

**Issue by somebrel** [steveklabnik/rustdoc#44]
In [`CONTRIBUTING.md`](https://github.com/steveklabnik/rustdoc/blob/master/CONTRIBUTING.md#frontend-development-workflow) we say run
```shell
> cd frontend
> npm install
> ember serve
> ...
```
but `ember serve` won't work for users that don't have ember-cli installed globally. `npm install` installs dependencies locally.

We could tell the developers to
1. install ember globally: `npm install -g ember-cli`
2. use the local copy of ember-cli installed by npm. That is, to use `./node_modules/ember-cli/bin/ember serve` instead of `ember serve`
3. use npm scripts

something else?

I think that the best option is to use **npm scripts** because some developers maybe don't want (or can't) go for option 1 and option 2 is too long. And If we look in `frontend/package.json`, we'll see that we even have a script for `ember serve`
```json
...
  "scripts": {
    "build": "ember build",
    "start": "ember server",
    "test": "ember test",
    "eslint": "eslint app"
  },
...
```
In this case we'll have to add one script for `ember build -prod` e.g. `build-prod` and  then, in [`CONTRIBUTING.md`](https://github.com/steveklabnik/rustdoc/blob/master/CONTRIBUTING.md#frontend-development-workflow), change 

`ember serve` to `npm run start`or just `npm start ` 

and

`ember build -prod` to `npm run build-prod`

**Comment by bharatzvm** [comment]
Adding npm scripts makes sense

**Comment by steveklabnik** [comment]
👍 for npm scripts

**Comment by mgattozzi** [comment]
Should this be closed now that #45 was merged?

**Comment by steveklabnik** [comment]
yes!
